File: ASEDINFO_FRONTEND/src/app/main/pages/compartidos/modelos/InicioSesion.ts

import { get, set } from 'lodash-es';
import { IinicioSesion } from '../interfaces/IinicioSesion';
export class InicioSesion implements IinicioSesion {

    constructor(data) {
        set(this, 'data', data);
    }

    get codigo(): number {
        return get(this, 'data.codigo');
    }
    set codigo(value: number) {
        set(this, 'data.codigo', value);
    }

    get ip(): string {
        return get(this, 'data.ip');
    }
    set ip(value: string) {
        set(this, 'data.ip', value);
    }

    get usuario(): string {
        return get(this, 'data.usuario');
    }
    set usuario(value: string) {
        set(this, 'data.usuario', value);
    }

    get prefijoAplicacion(): string {
        return get(this, 'data.prefijoAplicacion');
    }
    set prefijoAplicacion(value: string) {
        set(this, 'data.prefijoAplicacion', value);
    }

    get fecha(): string {
        return get(this, 'data.fecha');
    }
    set fecha(value: string) {
        set(this, 'data.fecha', value);
    }

}

File: ASEDINFO_BACKEND/asedinfo-modelo/src/main/java/ec/gob/educacion/modelo/seguridad/InicioSesion.java
package ec.gob.educacion.modelo.seguridad;

import java.util.Date;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

import com.fasterxml.jackson.annotation.JsonFormat;

@Entity
@Table(name = "seg_inicio_sesion")
public class InicioSesion implements java.io.Serializable {
	private static final long serialVersionUID = 1L;

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@Column(name = "codigo", unique = true, nullable = false, precision = 10, scale = 0)
	private Long codigo;

	@Column(name = "ip")
	private String ip;
	
	@Column(name = "usuario")
	private String usuario;
	
	@Column(name = "prefijo_aplicacion")
	private String prefijoAplicacion;
	
	@Temporal(TemporalType.TIMESTAMP)
	@Column(name = "fecha", length = 23)
	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd HH:mm:ss.SSS", locale = "es-EC", timezone = "America/Lima")
	private Date fecha;

	public InicioSesion() {
	}

	public Long getCodigo() {
		return this.codigo;
	}

	public void setCodigo(Long codigo) {
		this.codigo = codigo;
	}

	public String getIp() {
		return ip;
	}

	public void setIp(String ip) {
		this.ip = ip;
	}

	public String getUsuario() {
		return usuario;
	}

	public void setUsuario(String usuario) {
		this.usuario = usuario;
	}

	public String getPrefijoAplicacion() {
		return prefijoAplicacion;
	}

	public void setPrefijoAplicacion(String prefijoAplicacion) {
		this.prefijoAplicacion = prefijoAplicacion;
	}

	public Date getFecha() {
		return fecha;
	}

	public void setFecha(Date fecha) {
		this.fecha = fecha;
	}
}
